books/RayTracingInOneWeekend.html

```diff
@@ -1489,7 +1490,11 @@
         }
 
         ray get_ray(double s, double t) const {
-            return ray(origin, lower_left_corner + s*horizontal + t*vertical - origin);
+            // Return the ray from the projection point to the indicated pixel. Coordinates s,t are
```

Comment:
This needs to be in text. Not comments.

Reply:
I'd actually rather we did better commenting the code, instead of telling folks to "read the book". Particularly in cases here where we have an implicit change of coordinate systems. Before the change was sort of smeared across the call stack, but now at least we have a crisp point where the flip happens.